docs/topics/coroutines-and-channels.md

```diff
@@ -91,7 +91,10 @@ This API is used by the `loadContributorsBlocking()` function to fetch the list
 1. Open `src/tasks/Request1Blocking.kt` to see its implementation:
 
     ```kotlin
-    fun loadContributorsBlocking(service: GitHubService, req: RequestData): List<User> {
+    fun loadContributorsBlocking(
+    service: GitHubService,
+    req: RequestData
```

Comment:
Is the indentation here correct?

Reply:
No, this is a copy-paste issue from a file with code to the example area in the md file. Fixed it here https://github.com/Kotlin/kotlinx.coroutines/pull/4085/files